src/components/PriorAuth/PriorAuth.jsx

```diff
@@ -46,46 +46,39 @@ export default class PriorAuth extends Component {
     this.setState({
       subscriptionType: subscriptionType,
       showRestHookForm:
-        subscriptionType === this.subscriptionType.RESTHOOK ? true : false
+        subscriptionType === this.subscriptionType.RESTHOOK ? true : false,
     });
   }
 
   /**
    * Poll for a pended claim based on the IG specification
-   * Specification: no more than 4 times in first hour
+   * Specification: every 5 min for the first half hour
    *                no more than once per hour after that
    *                at least once every 12 hours
    */
   polling() {
-    // let hourInSec = 3600;
-    let hourInSec = 32; // For testing purposes make an hour 16 seconds
+    // const hourInSec = 3600;
+    const hourInSec = 32; // For testing purposes make an hour 16 seconds
```

Comment:
Is there a way to invoke the testing version via a config file or command line parameter?   I was a bit confused the first time I read this.

Reply:
Added a config file